django/thunderstore/core/storage.py

```diff
@@ -0,0 +1,16 @@
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+
+from thunderstore.utils.makemigrations import StubStorage, is_migrate_check
+
+# These are required as a placeholder stub for migrations, otherwise Django thinks
+# something keeps changing due to settings being different.
+
+
+def handle_storage_class_assignment():
+    if is_migrate_check():
+        return StubStorage()
+    return get_storage_class(settings.PACKAGE_FILE_STORAGE)
+
+
+DEFAULT_STORAGE_CLASS = handle_storage_class_assignment()
```

Comment:
My bad on the last review, this should probably be called `PACKAGE_STORAGE_CLASS`, or _alternatively_ we should remove the entire `PACKAGE_FILE_STORAGE` setting and just use the default storage here.

But this made me realize we could probably skip this change entirely & assign the stub storage to the `PACKAGE_FILE_STORAGE` setting during migration checks in the settings directly. That way we won't need to change anything in here.